icommons_ext_tools/settings/base.py

```diff
@@ -301,8 +274,8 @@
     'QUALTRICS_API_USER': SECURE_SETTINGS.get('qualtrics_api_user'),
     'QUALTRICS_API_TOKEN': SECURE_SETTINGS.get('qualtrics_api_token'),
     'QUALTRICS_AUTH_GROUP': SECURE_SETTINGS.get('qualtrics_auth_group'),
-    'USER_DECLINED_TERMS_URL': 'ql:internal',
-    'USER_ACCEPTED_TERMS_URL': 'ql:internal',
+    'USER_DECLINED_TERMS_URL': SECURE_SETTINGS.get('qualtrics_user_declined_terms_url'),
```

Comment:
I didn't realize these values were varying by environment.  Can you share what the urls should be in dev, qa, stage, and prod so I can get them into our secure settings files?  Feel free to Slack or email them to me.